contrib/dump_comments.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+#
+# The MIT License (MIT)
+#
+# Copyright (c) 2012-2014 Martin Zimmermann.
```

Comment:
This should probably have your name rather than Martin's :-)

Reply:
Fixed !